common.py

```diff
@@ -354,48 +354,41 @@ def initialize_theta(theta_shape, possible_theta_mask, zero_theta_mask):
     theta[zero_theta_mask] = 0
     return theta
 
-def split_train_val(num_obs, metadata, tuning_sample_ratio, validation_column=None, val_column_idx=None):
+def create_theta_idx_mask(zero_theta_mask_refit, possible_theta_mask):
     """
-    @param num_obs: number of observations
-    @param metadata: metadata to include variables to perform validation on
-    @param tuning_sample_ratio: ratio of data to place in validation set
-    @param validation_column: variable to perform validation on (if None then sample randomly)
-    @param val_column_idx: which index to pick for K-fold validation
-
-    @return training and validation indices
+    From an aggregate theta, creates a matrix with the index of the hierarchical theta
     """
-    if validation_column is None:
-        # For no validation column just sample data randomly
-        val_size = int(tuning_sample_ratio * num_obs)
-        if tuning_sample_ratio > 0:
-            val_size = max(val_size, 1)
-        permuted_idx = np.random.permutation(num_obs)
-        train_idx = permuted_idx[:num_obs - val_size]
-        val_idx = permuted_idx[num_obs - val_size:]
+    theta_idx_counter = np.ones(possible_theta_mask.shape, dtype=int) * -1
+    theta_mask = ~zero_theta_mask_refit & possible_theta_mask
+    idx = 0
+    for col in range(theta_mask.shape[1]):
+        for row in range(theta_mask.shape[0]):
+            if theta_mask[row, col]:
+                theta_idx_counter[row, col] = idx
+                idx += 1
+    return theta_idx_counter
+
+def create_aggregate_theta(hier_feat_generator, agg_feat_generator, theta, zero_theta_mask, possible_theta_mask, keep_col0=True, add_targets=True):
```

Comment:
@dawahs -- this function is used by other files, though you deleted it in your previous PR... i also noticed that this function is using `combine_thetas_and_get_conf_int` not correctly. can you help fix this up? either in this PR or a separate one i guess?